README.md

```diff
@@ -27,35 +29,74 @@ expirationd.start(job_name, space.id, is_expired, {
 * `name` - task name
 * `space_id` - space to look in for expired tuples
 * `is_tuple_expired` - a function, must accept tuple and return true/false
   (is tuple expired or not), receives `(args, tuple)` as arguments
 opt
 * `options` -- (table with named options, may be nil)
-  * `process_expired_tuple` - applied to expired tuples, receives `(space_id, args, tuple)`
-    as arguments. Can be nil: by default tuples are removed
-  * `args` - passed to `is_tuple_expired()` and `process_expired_tuple()` as additional context
-  * `tuples_per_iteration` - number of tuples will be checked by one iteration
-  * `full_scan_time` - time required for full index scan (in seconds)
-  * `iteration_delay` - max sleep time between iterations (in seconds)
-  * `full_scan_delay` - sleep time between full scans (in seconds)
-  * `on_full_scan_start` - call function on starting full scan iteration
-    Receives `(task)` as arguments.
-  * `on_full_scan_complete` - call function on complete full scan iteration.
-    Called after `on_full_scan_success` or `on_full_scan_error`.
-    Receives `(task)` as arguments.
-  * `on_full_scan_success` - call function on success full scan iteration
-    Receives `(task)` as arguments.
-  * `on_full_scan_error` - call function on error full scan iteration
-    Receives `(task, error)` as arguments.
-  * `force` - run, even on replica
+    * `process_expired_tuple` - Applied to expired tuples, receives (space_id, args, tuple) as arguments. Can be nil: by default, tuples are removed.
+    * `index` - Name of the index to iterate on. If omitted, will use the primary index. If there's no index with this name, will throw an error. supported index types are TREE and HASH, using other types will result in an error.
+    * `iterator_type` - Type of the iterator to use, as string or box.index constant, for example, 'EQ' or box.index.EQ. Default is box.index.ALL. See https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_index/pairs/.
+    * `start_key` - Start iterating from the tuple with this index value. Or when iterator is 'EQ', iterate over tuples with this index value. Must be a value of the same data type as the index field or fields, or a function which returns such value. If omitted or nil, all tuples will be checked.
+    * `tuples_per_iteration` - Number of tuples to check in one batch (iteration). Default is 1024.
+    * `atomic_iteration` - Boolean, false (default) to process each tuple as a single transaction. True to process tuples from each batch in a single transaction.
+    * `process_while` - Function to call before checking each tuple. If it returns false, the task will stop until next full scan.
+    * `iterate_with` - Function which returns an iterator object which provides tuples to check, considering the start_key, process_while and other options. There's a default function which can be overriden with this parameter.
+    * `on_full_scan_start` - Function to call before starting a full scan iteration.
```

Comment:
before starting a tuple scan. 

I'd rather avoid mentioning "full scan" here and further, since it is not necessary full scan any more.